int64 PointNum
float64[16] PosX
float64[16] PosY
float64[16] PosZ